pkg/kubelet/kubelet.go

```diff
@@ -1386,7 +1386,11 @@ func (kl *Kubelet) GetKubeletContainerLogs(podFullName, containerName, tail stri
 		// waiting state.
 		return err
 	}
-	return kl.containerManager.GetContainerLogs(containerID, tail, follow, stdout, stderr)
+	pod, ok := kl.GetPodByFullName(podFullName)
```

Comment:
Is it still necessary to get the pod status and perform various checks above? It feels like those could be redundant after your change.